src/style.css

```diff
@@ -0,0 +1,62 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.menu__list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+}
+
+.menu__link {
+  text-decoration: none;
+  display: block;
+  position: relative;
+  padding: 23px 0;
+  margin-left: 20px;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  text-transform: uppercase;
+  color: #000;
+}
+
+.first_link {
+  margin: 0;
+}
```

Comment:
you can use pseudo-class :first-child here, instead of creating an extra class

Reply:
if i do that all the links lose their margin cuz they are all first children in <li> tags